caikit/interfaces/runtime/data_model/info.py

```diff
@@ -39,3 +39,30 @@ class RuntimeInfoRequest(DataObjectBase):
 class RuntimeInfoResponse(DataObjectBase):
     runtime_version: Annotated[Optional[str], FieldNumber(1)]
     python_packages: Annotated[Dict[str, str], FieldNumber(2)]
+
+
+@dataobject(RUNTIME_PACKAGE)
+class ModelInfoRequest(DataObjectBase):
+    """Empty request for runtime server information"""
```

Comment:
Does this need to specify a specific `model_id`?

Reply:
Good call on the list! Updating the endpoint/tests now